raytracing/imagingpath.py

```diff
@@ -585,6 +585,9 @@ def imageSize(self):
         """
         fieldOfView = self.fieldOfView()
         (distance, conjugateMatrix) = self.forwardConjugate()
```

Comment:
Should we check distance == +inf or matrix is None?

Reply:
I am also torn between Matrix == None or a matrix with infinite element B. Not sure what to do here or what it implies to have a matrix with +inf in the calculations.